### PR TITLE
feat: Update dependencies

### LINKED ISSUE #145: 6.1.0 seems to have a regression
- `kcd-scripts` version: 6.1.0
- `node` version: 12 & 14
- `npm` (or `yarn`) version: latest

Relevant code or config

https://github.com/testing-library/testcafe-testing-library/pull/167

```javascript
Validate
```

What you did

Dependabot tried to update to 6.1.0
What happened:
Something broke between 6.0.1 & 6.1.0 not sure what but validate is breaking on testcafe testing library.

Reproduction repository:

https://github.com/testing-library/testcafe-testing-library/pull/167

Problem description:

Suggested solution:

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Closes #145

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "@rollup/plugin-json": "^4.0.3",
     "@rollup/plugin-node-resolve": "^8.0.0",
     "@rollup/plugin-replace": "^2.3.2",
+    "@types/jest": "^25.2.3",
```
Comment (kentcdodds) [inline package.json:48]: Ah, good idea!

File: package.json [hunk 2]
```diff
@@ -76,11 +77,11 @@
     "read-pkg-up": "^7.0.1",
     "resolve": "^1.17.0",
     "rimraf": "^3.0.2",
-    "rollup": "^2.10.5",
+    "rollup": "^2.10.9",
     "rollup-plugin-node-builtins": "^2.1.2",
     "rollup-plugin-node-globals": "^1.4.0",
-    "rollup-plugin-size-snapshot": "^0.11.0",
-    "rollup-plugin-terser": "^5.3.0",
+    "rollup-plugin-size-snapshot": "^0.12.0",
```
Comment (kentcdodds) [inline package.json:83]: Kinda want to remove this feature in the next major version we need to make.

File: package.json (hunk 1 above)
Comment (MichaelDeBoey) [reply package.json:48]: I had removed it, since `jest` now has its own types, but seems like that caused #145 😓

**Pull request merged**